lib/scala/project-manager/src/main/scala/org/enso/projectmanager/requesthandler/RequestHandler.scala

```diff
@@ -137,3 +176,7 @@ abstract class RequestHandler[
     context.become(responseStage(id, replyTo, None))
   }
 }
+
+object RequestHandler {
+  val defaultNumOfTimeoutRetries: Int = 3
```

Comment:
I believe this belongs to the [`application.conf`](https://github.com/enso-org/enso/blob/ebee8700ce436218613602c68f29ea826e791634/lib/scala/project-manager/src/main/resources/application.conf). The config is created in the `ProjectManager`
https://github.com/enso-org/enso/blob/ebee8700ce436218613602c68f29ea826e791634/lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/ProjectManager.scala#L35-L40
and it should not be that hard to propagate the value to the `RequestHandler`

Reply:
Good suggestion, I will change that